.travis.yml

```diff
@@ -8,3 +8,6 @@ addons:
   apt:
     packages:
       - libmagick++-dev
+
+bioc_packages:
```

Comment:
@hadley, 
I just realized that `BiocInstaller` needs to be available on Travis-CI to be able to use `requireNamespace`. I can either remove all mention of `BiocInstaller` and leave hard-coded links or include `BiocInstaller` in the `Suggests` field and on Travis-CI. Thoughts?